pages/best-practices/avo-functions-alongside-existing-tracking.mdx

```diff
@@ -0,0 +1,214 @@
+
+First we import the Avo Functions and define an interface for our Custom Destination, where we connect it to our analytics SDK:
+
+```typescript
+import Avo from './Avo';
+
+// An interface for your Custom Destination that you pass into initAvo():
+let myAnalyticsSDKBridge = {
+  make: function (env) {
+    /*
+      Triggered during Avo initialization. Here you can initialize an analytics SDK with a development or production key
+      based on the env parameter. If analytics SDK has already been initialized you can leave this function blank.
+
+      Example: analytics.init(env);      
+    */
+  },
+  logEvent: (eventName, eventProperties) => {
+    /*
+      All your events are managed in the Tracking Plan in Avo. Each event gets a generated Avo Function. Avo events can
+      have a Log Event action attached. This callback is triggered when an Avo Function with Log Event action is called.
+      Here you perform the actual event tracking, calling the track/log methods of the analytics SDK. Event name and event
+      properties are provided as parameters.
+
+      Example: analytics.track(eventName, eventProperties);  
+    */
+  },
+  setUserProperties: (userId, userProperties) => {
+    /*
+      You can add User Properties to events in the Trackin Plan in Avo. When an Avo Function with attached user properties
+      is called, this callback is triggered. Here you would attach user properties to the currently identified user in your
+      analytics platform.
+
+      Example: analytics.setUserProperties(userId, userProperties);  
+    */
+  },
+  identify: (userId) => {
+    /*
+      You can add the Identify User action to events in the Trackin Plan in Avo. When calling an Avo Function that includes
+      the Identify User action you'll need to provide a user ID. Calling an Avo Function that has the Identify User action
+      will trigger this callback. The main use cases are signup and login. Here you would pass the user ID to the analytics
+      SDK for it to create a new user or attach a session to an existing user.
+
+      Example: analytics.identify(userId);
+    */
+  },
+  unidentify: () => {
+    /*
+      You can add the Unidentify User action to events in the Trackin Plan in Avo When calling an Avo Function that includes
+      the Unidentify User action this callback will be triggered. Here you would call an analytics SDK method to detach subsequent
+      actions from the currently identified user.
+
+      Example: analytics.identify(null);
+    */
+  },
+};
+```
+
+Then we call the initAvo Function:
+
+```typescript
+Avo.initAvo({ env: Avo.AvoEnv.Dev }, {}, myAnalyticsSDKBridge);
+```
+
+<br />
+
+Detailed instructions for all other programming languages and platforms supported by Avo [can be found here](/custom-destinations#initialization-instructions).
```

Comment:
Would probably move `/custom-destinations#initialization-instructions` to the `Implement` section